packages/fiori/src/UploadCollectionItem.js

```diff
@@ -257,13 +257,13 @@ class UploadCollectionItem extends ListItem {
 	}
 
 	onBeforeRendering() {
-		if (!this.focused) {
```

Comment:
Is this file intended to be changed?

Reply:
> Is this file intended to be changed?

It is intended. This was the reason the PR was failing. From one hand we dont want the entire lis item to get the outline when we interact with ui5-button ui5-link and I fixed this, but the UploadCollection used to depend this.focused to turn on/off the editing mode, now it relies on its own this._focused.